base/float.jl

```diff
@@ -261,7 +261,7 @@ float(x) = convert(AbstractFloat, x)
     float(T::Type)
 
 Returns an appropriate type to represent a value of type `T` as a floating point value.
-Equivalent to `typeof(float(zero(T)))`.
+Falls back to `typeof(float(zero(T)))`.
```

Comment:
Technically "equivalent to" is still correct. For clarity I think I'd actually leave that how it was, since saying that it falls back to a particular implementation doesn't tell you anything about what it does in other cases.